test/unit/spec/connect.js

```diff
@@ -82,6 +82,12 @@ describe('connect', () => {
       value: true,
       shouldDelete: true
     },
+    {
+      name: 'eventListener',
```

Comment:
The name might be a bit confusing here, should this be called "loggerListener" or "logEntryListener"?

I'm assuming this is a listener for when the logger logs an event, as opposed to the deprecated interface which I believe was also called `eventListener`

Reply:
@idelgado  This test is verifying that we handle the deprecated 'eventListener' correctly. The name need to be `eventListener`